cmd/console/kics.go

```diff
@@ -0,0 +1,144 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"time"
+
+	"github.com/Checkmarx/kics/internal/storage"
+	"github.com/Checkmarx/kics/internal/tracker"
+	"github.com/Checkmarx/kics/pkg/engine"
+	"github.com/Checkmarx/kics/pkg/engine/query"
+	"github.com/Checkmarx/kics/pkg/kics"
+	"github.com/Checkmarx/kics/pkg/model"
+	"github.com/Checkmarx/kics/pkg/parser"
+	dockerParser "github.com/Checkmarx/kics/pkg/parser/docker"
+	terraformParser "github.com/Checkmarx/kics/pkg/parser/terraform"
+	yamlParser "github.com/Checkmarx/kics/pkg/parser/yaml"
+	"github.com/Checkmarx/kics/pkg/source"
+	"github.com/getsentry/sentry-go"
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
+)
+
+func startKics(ctx context.Context, path, queryPath, outputPath, payloadPath string, verbose, logFile, version bool) error {

```

Comment:
can we rename the function to **init**?

Reply:
I thought about that. If we change to another package, I think it should be okay, but if not, we can't since it will execute before we prepared Cobra with the parameters...